plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/CDKServerWizardFragment.java

```diff
@@ -67,6 +68,7 @@ public ImageDescriptor getImageDescriptor() {
 	@Override
 	public Composite createComposite(Composite parent, IWizardHandle handle) {
 		this.handle = handle;
+		selectedUser = null;
```

Comment:
I'm not sure what you mean by "assumes no sense", but if you mean "doesn't make sense", then you'd be wrong.  The WTP environment only creates one instance of the wizard fragment, and so any time it's used, its member variables basically need to be reset.